src/components/member/tabs/contract-tab/FromDateComponent.tsx

```diff
@@ -0,0 +1,71 @@
+import { Agreement, Contract } from 'api/generated/graphql'
+import { format } from 'date-fns'
+import { Button, ButtonsGroup } from 'hedvig-ui/button'
+import { DateTimePicker } from 'hedvig-ui/date-time-picker'
+import * as React from 'react'
+import { Card } from '../../../../../shared/hedvig-ui/card'
+import {
+  changeFromDateOptions,
+  useChangeFromDate,
+} from '../../../../graphql/use-change-from-date-for-agreement'
+import { ButtonSpacing } from './contract-item'
+
+const initialFromDate = (agreement: Agreement): Date =>
+  agreement.fromDate ? new Date(agreement.fromDate) : new Date()
+
+const FromDateComponent: React.FunctionComponent<{
+  agreement: Agreement
+  contract: Contract
+}> = ({ agreement, contract }) => {
+  const [datePickerEnabled, setDatePickerEnabled] = React.useState(false)
+  const [fromDate, setFromDate] = React.useState(initialFromDate(agreement))
+  const [
+    changeFromDate,
+    { loading: changeFromDateLoading },
```

Comment:
This is 100% subjective, but if we only have 1 loading in the component I prefer calling the variable `loading` since it's already pretty clear what is loading
```suggestion
    { loading },
```